docs/UserGuide.md

```diff
@@ -1,3 +1,13 @@
+## Quick start
+
+1. Ensure you have Java 11 or above installed in your Computer.
+2. Download the latest TABS.jar.
+3. Copy the file to the folder you want to use as the home folder for your TABS.
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
+![TABS example](Ui.png)
```

Comment:
Formatting is weird in preview. Maybe add line break between the line 7 (link) and 6?